cassandra-mesos-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/framework/Main.java

```diff
@@ -134,7 +136,8 @@ private static int _main() throws UnknownHostException {
         final long      healthCheckIntervalSec      = Long.parseLong(       Env.option("CASSANDRA_HEALTH_CHECK_INTERVAL_SECONDS").or("60"));
         final long      bootstrapGraceTimeSec       = Long.parseLong(       Env.option("CASSANDRA_BOOTSTRAP_GRACE_TIME_SECONDS").or("120"));
         final String    cassandraVersion            =                       "2.1.4";
-        final String    frameworkName               = frameworkName(        Env.option("CASSANDRA_CLUSTER_NAME"));
```

Comment:
I understand the change. I was actually referring to consistent coding style clusterNameOpt should not be a separate variable for consistency. It is a nit, but I would prefer to be as clean as possible.